Angela/tenant content strategy update (#3388)
cloudflare/cloudflare-docs: products/tenant/src/content/index.md

pcx-content-type: landing-page
---

# Cloudflare Tenant

> With Cloudflare’s new provisioning APIs, you can set up and manage Cloudflare accounts and services for your team or your customers. A new set of APIs built into our Client v4 API library make a streamlined onboarding and setup experience.

Reviewer: ```suggestion
With Cloudflare’s provisioning APIs, you can set up and manage Cloudflare accounts and services for your team or your customers. These APIs built into our Client v4 API library provide a streamlined onboarding and setup experience.
```

Reviewer: Need to be careful about `new` terminology persisting for long periods of time.
